apps/api/v2/src/modules/verified-resources/outputs/verified-phone.output.ts

```diff
@@ -0,0 +1,73 @@
+import { ScheduleOutput } from "@/ee/schedules/schedules_2024_04_15/outputs/schedule.output";
+import { ApiProperty } from "@nestjs/swagger";
+import { Expose, Type } from "class-transformer";
+import { IsArray, IsEnum, IsNotEmptyObject, IsNumber, IsString, ValidateNested } from "class-validator";
+
+import { ERROR_STATUS, SUCCESS_STATUS } from "@calcom/platform-constants";
+
+export class VerifiedPhoneOutputData {
+  @Expose()
+  @IsNumber()
+  @ApiProperty({
+    description: "The unique identifier for the verified email.",
+    example: 789,
+  })
+  id!: number;
+
+  @Expose()
+  @IsString()
+  @ApiProperty({
+    description: "The verified phone number.",
+    example: "+37255556666",
+    format: "phone",
+  })
+  phoneNumber!: string;
+
+  @Expose()
+  @IsNumber()
+  @ApiProperty({
+    description: "The ID of the associated user, if applicable.",
+    example: 45,
+    nullable: true,
+    required: false,
+  })
```

Comment:
Shall we have 2 classes here too 1 for user 1 for team so there is no optional props.